functorch/test/test_control_flow.py

```diff
@@ -0,0 +1,154 @@
+import torch
+
+from torch.testing._internal.common_utils import TestCase
+from functorch.experimental.cond import cond
+from torch.fx.experimental.proxy_tensor import make_fx
+
+
+class TestControlFlow(TestCase):
+    def test_cond(self):
+        def true_fn(x):
+            return x.sin()
+
+        def false_fn(x):
+            return x.cos()
+
+        x = torch.randn(4)
+        result = cond(False, true_fn, false_fn, x)
+        self.assertTrue(torch.allclose(result, torch.cos(x)))
```

Comment:
nit: self.assertEqual is the "blessed" way of doing this. self.assertEqual will actually print out the tensors if it fails